machine-learning/03-Training and AutoML.ipynb

```diff
@@ -1,45 +1,561 @@
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Loading extensions from `MLNetAutoML.InteractiveExtension.dll`"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Loading extensions from `XPlot.Plotly.Interactive.dll`"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Configuring PowerShell Kernel for XPlot.Plotly integration."
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Installed support for XPlot.Plotly."
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Loading extensions from `Microsoft.Data.Analysis.Interactive.dll`"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "// install dependencies and import using statement\n",
+        "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json\" \n",
+        "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json\"\n",
+        "#i \"nuget:https://mlnetcli.blob.core.windows.net/mlnetcli/index.json\"\n",
+        "\n",
+        "// add nightly build for ml.net\n",
+        "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/MachineLearning/nuget/v3/index.json\"\n",
+        "\n",
+        "#r \"nuget:MLNetAutoML.InteractiveExtension,0.1.1\"\n",
+        "#r \"nuget:XPlot.Plotly.Interactive,4.0.6\"\n",
+        "#r \"nuget:Microsoft.ML.AutoML,0.20.0-preview.22226.2\"\n",
+        "#r \"nuget:Microsoft.Data.Analysis,0.20.0-preview.22226.2\"\n",
+        "\n",
+        "// Import usings.\n",
+        "using Microsoft.Data.Analysis;\n",
+        "using System;\n",
+        "using System.IO;\n",
+        "using Microsoft.ML;\n",
+        "using Microsoft.ML.AutoML;\n",
+        "using Microsoft.ML.Data;\n",
+        "using MLNetAutoML.InteractiveExtension;"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Create linear dataset\n",
+        "The code below artifacts linear dataset with a random residual, and loaded it as train/test `DataFrame`"
```

Comment:
```suggestion
        "The code below creates a linear dataset with a random residual. The dataset is loaded into train and test DataFrames"
```

Reply:
The result of TrainTestSplit is an IDataView, so I think we might have to change the wording from DataFrames to IDataView. Or we can just remove it and say data is split into train and test sets.